### PR TITLE
refactor: improvement of the French translation

### PR DESCRIPTION
**Pull request by MoskalykA**

### CODE REVIEW

File: langs/fr.json [hunk 1]
```diff
@@ -3,103 +3,178 @@
...
     "DIRECT_CONNECT": "Connexion directe",
     "ABOUT": "À propos",
-    "DISCONNECT": "Déconnexion",
+    "DISCONNECT": "Déconnecter",
     "SETTINGS": "Paramètres",
     "EXIT": "Quitter",
     "SEARCH": "Rechercher",
     "HIDE": "Cacher",
-    "FULL": "Complet",
+    "FULL": "Plein",
```
Comment (yannbcf) [inline langs/fr.json:23]: "Complet" is better in this context

File: langs/fr.json [hunk 2]
```diff
@@ -3,103 +3,178 @@
...
     "FAVORITE_SERVERS": "Serveurs favoris",
     "LANGUAGE": "Langue",
     "ENTER_PASSWORD": "Entrer le mot de passe",
-    "CONNECT": "Connexion",
-    "NICKNAME": "Surnom",
+    "CONNECT": "Se connecter",
+    "NICKNAME": "Pseudo",
     "SERVERLIST": "Liste des serveurs",
-    "PROMOTED_ON_TOP": "Serveurs mis en avant en premier",
+    "PROMOTED_ON_TOP": "Serveurs promus en haut de la liste",
```
Comment (yannbcf) [inline langs/fr.json:37]: "Serveurs mis en avant" is better

File: langs/fr.json [hunk 3]
```diff
@@ -3,103 +3,178 @@
...
-    "SERVERS_ONLINE": "Serveurs en ligne: {0}",
+    "PLAYERS_ONLINE:ONE": "{0} joueur en ligne",
+    "PLAYERS_ONLINE:OTHER": "{0} joueurs en ligne",
+    "SERVERS_ONLINE:ONE": "{0} serveur en ligne",
+    "SERVERS_ONLINE:OTHER": "{0} serveurs en ligne",
     "SERVERS": "Serveurs",
     "FAVORITES": "Favoris",
     "HISTORY": "Historique",
-    "PROMOTED": "Mis en avant",
+    "PROMOTED": "Promus",
```
Comment (yannbcf) [inline langs/fr.json:15]: This change is a regression to me

File: langs/fr.json [hunk 4]
```diff
@@ -3,103 +3,178 @@
...
-    "EARLY_AUTH_FAILED": "L'autorisation précoce a échoué",
-    "HTTP_SERVER_CONNECT_FAILED": "La connexion au serveur HTTP a échoué",
-    "RESOURCE_DOWNLOAD_FAILED": "Le téléchargement de la ressource a échoué",
-    "RESOURCE_CHECK_FAILED": "Le contrôle d'intégrité de la ressource a échoué",
-    "CHANGE_NICKNAME": "Changer votre surnom",
+    "EARLY_AUTH_FAILED": "Authentification précoce échouée",
+    "HTTP_SERVER_CONNECT_FAILED": "Impossible de se connecter au serveur HTTP",
+    "RESOURCE_DOWNLOAD_FAILED": "Échec du téléchargement de la ressource",
+    "RESOURCE_CHECK_FAILED": "La vérification d'intégrité de la ressource a échoué",
+    "CHANGE_NICKNAME": "Changer votre pseudonyme",
```
Comment (yannbcf) [inline langs/fr.json:64]: "pseudo" seems better

File: langs/fr.json [hunk 5]
```diff
@@ -3,103 +3,178 @@
...
+    "WRONG_BETA_BUILD": "Le serveur utilise une version plus récente d'alt:V. Redémarrez votre jeu pour mettre à jour.",
+    "CLIENT_OUTDATED": "Le serveur utilise une version plus récente d'alt:V",
+    "SERVER_OUTDATED": "Le serveur utilise une version plus ancienne d'alt:V",
     "REFUSED_BY_SERVER": "Refusé par le serveur",
-    "EARLY_AUTH_FAILED": "L'autorisation précoce a échoué",
-    "HTTP_SERVER_CONNECT_FAILED": "La connexion au serveur HTTP a échoué",
-    "RESOURCE_DOWNLOAD_FAILED": "Le téléchargement de la ressource a échoué",
-    "RESOURCE_CHECK_FAILED": "Le contrôle d'intégrité de la ressource a échoué",
-    "CHANGE_NICKNAME": "Changer votre surnom",
+    "EARLY_AUTH_FAILED": "Authentification précoce échouée",
```
Comment (yannbcf) [inline langs/fr.json:60]: Not directly related to the changes but "précoce" as a literal "early" translation is very weird

File: langs/fr.json [hunk 6]
```diff
@@ -3,103 +3,178 @@
...
-    "INVALID_CDN_URL": "Le cdn ne correspond pas au cdn du serveur",
-    "DEBUG": "Debug",
-    "USE_EXTERNAL_CONSOLE": "Ouvrir la console dans une fenêtre séparée",
-    "SERVER_DATA_SETTINGS": "Paramètres des informations locales du serveur",
-    "DELETE_RESOURCES": "Supprimer les ressources du serveur",
-    "DELETE_DATA": "Supprimer les informations locales du serveur",
-    "CONFIRM_RESOURCE_DELETE": "Êtes-vous sûr de vouloir supprimer {0} de ressources du serveur?\nVous devrez les télécharger à nouveau lors de votre prochaine connexion",
-    "CONFIRM_DATA_DELETE": "Êtes-vous sûr de vouloir supprimer {0} d'informations locales du serveur?\nCela peut inclure les nom d'utilisateur, mot de passe, paramètres, etc",
+    "INVALID_CDN_URL": "Le CDN ne correspond pas au CDN du serveur",
+    "DEBUG": "Déboguer",
```
Comment (yannbcf) [inline langs/fr.json:85]: I guess keeping "debug" is better as we keep "crash" instead of "accident"

**Review by yannbcf — CHANGES_REQUESTED** [review]

**Comment by MoskalykA** [comment]
@yannbcf 👍🏻

**Comment by C0kkie** [comment]
Thanks for contrib

**Pull request merged**